src/lib/seam/components/AccessCodeDetails/AccessCodeDevice.tsx

```diff
@@ -56,7 +56,8 @@ function Content(props: {
         <DeviceImage device={device} />
       </div>
       <div className='seam-body'>
-        <div className='seam-model'>
+        <div>{device.properties.name}</div>
+        <div>
           {device.properties.model.manufacturer_display_name}{' '}
           {device.properties.model.display_name}
```

Comment:
We should remove ln 61-62

Reply:
[`df2739e` (#369)](https://github.com/seamapi/react/pull/369/commits/df2739e17598f8b4d3f8876e29e1fce15a11392d)